agents/craftassist/tests/test_default_behaviors.py

```diff
@@ -0,0 +1,24 @@
+import unittest
+from agents.craftassist.tests.base_craftassist_test_case import BaseCraftassistTestCase
+from droidlet.interpreter.craftassist.default_behaviors import build_random_shape, come_to_player
+
+
+class TestDefaultBehavior(BaseCraftassistTestCase):
+    def setUp(self):
+        super().setUp()
+
+    def test_build_random_shape(self):
+        schematic = build_random_shape(self.agent)
+        # Assert that some non-zero size schematic was built
+        self.assertTrue(len(schematic) > 0)
+        # Assert that build task was pushed and completed successfully
+        self.assertTrue(self.agent.memory.get_last_finished_root_task().action_name, "build")
+
+    def test_come_to_player(self):
+        come_to_player(self.agent)
+        # Assert that a move task was pushed and executed succesfully
+        self.assertTrue(self.agent.memory.get_last_finished_root_task().action_name, 'move')
```

Comment:
One thing to flag here @aszlam is that, after running the unit test for `come_to_player`, I tried:
```
from droidlet.lowlevel.minecraft.mc_util import pos_to_np

target = pos_to_np(random.choice(self.agent.get_other_players()))
manhat_dist(tuple(self.agent.pos), target)
```
and the `manhat_dist` is about 10 (not less than `approx` which is 3 [here](https://github.com/facebookresearch/droidlet/blob/main/droidlet/interpreter/craftassist/default_behaviors.py#L57))

Can you confirm if this is expected ?

Reply:
fixed in commit- there were two bugs here.  1: not flushing the task, 2: get_last_finished_root_task() was counting all tasks as finished if they were started early enough